### PR TITLE
Package xlsx2csv.0.1.2

### PR DESCRIPTION
**Pull request by zazedd**
### `xlsx2csv.0.1.2`
Simple library to export .xlsx files as CSVs
Simple library to export .xlsx files as CSVs

---
* Homepage: https://github.com/zazedd/xlsx2csv
* Source repo: git+https://github.com/zazedd/xlsx2csv.git
* Bug tracker: https://github.com/zazedd/xlsx2csv/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

File: packages/xlsx2csv/xlsx2csv.0.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+synopsis: "Simple library to export .xlsx files as CSVs"
+description: "Simple library to export .xlsx files as CSVs"
+maintainer: ["leomendesantos@gmail.com"]
+authors: ["Leonardo Santos"]
+license: "GPLv3"
+tags: ["convert" "xlsx" "csv"]
+homepage: "https://github.com/zazedd/xlsx2csv"
+bug-reports: "https://github.com/zazedd/xlsx2csv/issues"
+depends: [
+  "ocaml"
```
Comment (mseri) [inline packages/xlsx2csv/xlsx2csv.0.1.2/opam:12]:
```suggestion
  "ocaml" {>= "4.13"}
```
due to
```

#=== ERROR while compiling xlsx2csv.0.1.2 =====================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/zazedd/xlsx2csv/archive/refs/tags/0.1.2.tar.gz)
# path                 ~/.opam/4.12/.opam-switch/build/xlsx2csv.0.1.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p xlsx2csv -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/xlsx2csv-7-cc27a4.env
# output-file          ~/.opam/log/xlsx2csv-7-cc27a4.out
### output ###
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.xlsx2csv.objs/byte -I /home/opam/.opam/4.12/lib/astring -I /home/opam/.opam/4.12/lib/bytes -I /home/opam/.opam/4.12/lib/camlzip -I /home/opam/.opam/4.12/lib/fpath -I /home/opam/.opam/4.12/lib/lwt -I /home/opam/.opam/4.12/lib/lwt/unix -I /home/opam/.opam/4.12/lib/ocaml/threads -I /home/opam/.opam/4.12/lib/ocplib-endian -I /home/opam/.opam/4.12/lib/ocplib-endian/bigstring -I /home/opam/.opam/4.12/lib/uutf -I /home/opam/.opam/4.12/lib/xmlm -I /home/opam/.opam/4.12/lib/zip -no-alias-deps -open Xlsx2csv__ -o src/.xlsx2csv.objs/byte/xlsx2csv__Operations.cmo -c -impl src/operations.ml)
# File "src/operations.ml", line 14, characters 4-20:
# 14 |     String.fold_left
#          ^^^^^^^^^^^^^^^^
# Error: Unbound value String.fold_left
```

File: packages/xlsx2csv/xlsx2csv.0.1.2/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple library to export .xlsx files as CSVs"
+description: "Simple library to export .xlsx files as CSVs"
+maintainer: ["leomendesantos@gmail.com"]
+authors: ["Leonardo Santos"]
+license: "GPLv3"
```
Comment (mseri) [inline packages/xlsx2csv/xlsx2csv.0.1.2/opam:7]:
```suggestion
license: "GPL-3.0-only"
```
if I am not mistaken, can you double check?

Comment (zazedd) [reply packages/xlsx2csv/xlsx2csv.0.1.2/opam:7]: That seems to have fixed it, thanks!

**Comment by zazedd** [comment]
Sorry for the noobie question, it is the first time I am working with `opam-publish`, is there a way to update this PR using it? Or should I create a new tag and `opam publish` it again?

Thanks in advance.

**Comment by mseri** [comment]
If you re-tag and re-run opam publish, it should force push over this PR.

However, for these metadata change, you don't need to make a new release: opam will use the content of this opam file for the build, so the sources can have the old one without any problem. You only need to modify this in your main branch for future releases.

So, as long as you did not change the tag, I can actually make the changes here and merge, without any further action (besides updating your main branch with the same changes).

**Comment by mseri** [comment]
Thanks!

**Comment by mseri** [comment]
Feel free to announce the release to https://discuss.ocaml.org/, we have a community category and an announcement tag for this

**Comment by zazedd** [comment]
Thank you for the help!

**Pull request merged**